src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -35,10 +35,10 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
         return districtResult.partyResults;
     };
 
-    highLightWinnerRow = (winner: string): ComponentPropsGetter0 | ComponentPropsGetterR | undefined => {
+    highLightWinnerRow = (winner: string): ComponentPropsGetter0 | undefined => {
         return (state: undefined, rowInfo: RowInfo | undefined) => {
             if (rowInfo && rowInfo.original.partyCode === winner) {
-                return { className: "has-background-success" };
+                return { className: "input is-success" };
```

Comment:
Why are we using the "input" class here?

Reply:
To color the row-border. Might not be the proper way to do it.

Reply:
Can you describe the problem with images? What this looks like before, and what it fixes? It is likely not the correct approach regardless, given that it uses a non-intuitive class to get it done.

Reply:
`return { className: "has-background-success" };`
![background](https://user-images.githubusercontent.com/23212478/81662421-d246e980-943d-11ea-9cde-d3a406e4665b.png)
` return { className: "input is-success" };`
![border](https://user-images.githubusercontent.com/23212478/81662430-d6730700-943d-11ea-9ec9-4c22eec6e3c0.png)

Reply:
Do we know why the number is strangely offset in the second picture? That looks broken. Probably not using centering like the other cells are?

Regardless, I would prefer the upper alternative but with light text (so that it is readable). Sadly React-Table (at least the version we are on) has its own weird CSS setup for the table, which limits our options -- and makes Bulma less intuitive with use on the tables. There exist actual table classes for things like this:

https://bulma.io/documentation/elements/table/ is-selected for instance. If that works despite this not being a real table, it might make it easier.

Reply:
Seems like "is-selected" only work on <tr>.

Reply:
`return { className: "has-text-weight-bold" };`
![has-text-weight-bold](https://user-images.githubusercontent.com/23212478/81839846-68673680-9548-11ea-97ce-8870fc9884c4.png)
` return { className: "has-background-success has-text-white" };`
![has-text-white](https://user-images.githubusercontent.com/23212478/81839738-3e157900-9548-11ea-9525-01eaafcb6912.png)

`return { className: "has-text-success has-text-weight-bold" };`
![has-text-weight-bold-green](https://user-images.githubusercontent.com/23212478/81839592-0eff0780-9548-11ea-9b79-cde0c05d4af1.png)

Reply:
What about using a the background colour, but a different colour than success? If the colour was less intrusive I wouldn't feel it was any problem.

Reply:
Both the grey light and lighter look good to me. What do you think @simendag?

Reply:
I suggested the lighter one, so sure! With the new icon however, I think it might be unnecessary and place too much attention on the "last won" seat in the wrong places.